modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/web/internal/display/context/test/SegmentsExperienceSelectorDisplayContextTest.java

```diff
@@ -227,6 +228,12 @@ private JSONObject _getSegmentsExperienceSelectedJSONObject(
 			"segmentsExperienceId",
 			String.valueOf(selectedSegmentsExperienceId));
 
+		if (selectedSegmentsExperienceId != -1) {
```

Comment:
We should not add if statements to the tests, if we add them, then we should move this code to a Util class and test the behaviour.

Reply:
You should change the test 

testGetSegmentsExperienceSelectedJSONObject() and divide it to 2 different tests, one with selectedSegmentsExperienceId and another with selectedSegmentsExperienceId = -1.

Only one thing should be tested in one test: https://howtodoinjava.com/best-practices/unit-testing-best-practices-junit-reference-guide/#1